Sources/Parsing/Documentation.docc/Articles/Parsers/CharacterSet.md

```diff
@@ -5,7 +5,7 @@ A parser that consumes the characters contained in a `CharacterSet` from the beg
 For example:
 
 ```swift
-Parse {
+try Parse {
```

Comment:
This can actually be called without `try` because `CharacterSet` is a [non-throwing](https://github.com/pointfreeco/swift-parsing/blob/f22452a0e2e96d195a6ef5bfa1b8f91c0aa02470/Sources/Parsing/Parsers/CharacterSet.swift#L5) parser.

```suggestion
Parse {
```